post-scan-actions/aws-python-conformity-custom-check/README.md

```diff
@@ -0,0 +1,57 @@
+# Cloud One File Storage Security Post Scan Action - Conformity Custom Check
+
+After a scan occurs and a malicious file is detected, this example Lambda function creates a custom check within Cloud One Conformity.
+
+## Prerequisites
+
+1. **Configure Conformity API Key**
+    - Log into Conformity and select your username and select `User settings`.
+    - Select `API Keys`.
+    - Select `+ New API Key`, then select `Generate Key`.
+    - Copy the generated API Key for use later when installing the function.
+2. **Find the 'ScanResultTopic' SNS topic ARN**
+    - In the AWS console, go to **Services > CloudFormation** > your all-in-one stack > **Resources** > your storage stack > **Resources**.
+    - Scroll down to locate the  **ScanResultTopic** Logical ID.
+    - Copy the **ScanResultTopic** ARN to a temporary location. Example: `arn:aws:sns:us-east-1:123445678901:FileStorageSecurity-All-In-One-Stack-StorageStack-1IDPU1PZ2W5RN-ScanResultTopic-N8DD2JH1GRKF`
```

Comment:
Please also follow this change.
https://github.com/trendmicro/cloudone-filestorage-plugins/commit/2a13c9d112b4e81c5a96609ed869918bc69f3985
```suggestion
2. **Find the 'ScanResultTopicARN' SNS topic ARN**
    - In the AWS console, go to **Services > CloudFormation** > your all-in-one stack > **Outputs**  or **Services > CloudFormation** > your storage stack > **Outputs**.
    - Scroll down to locate the  **ScanResultTopicARN** Key.
    - Copy the **ScanResultTopic** ARN to a temporary location. Example: `arn:aws:sns:us-east-1:123445678901:FileStorageSecurity-All-In-One-Stack-StorageStack-1IDPU1PZ2W5RN-ScanResultTopic-N8DD2JH1GRKF`
```

Reply:
Done + squashed